ui/js/controllers/jobs.js

```diff
@@ -220,6 +220,28 @@ treeherderApp.controller('ResultSetCtrl', [
             });
         };
 
+        $scope.triggerAllTalosJobs = function(revision) {
+            if (!window.confirm('This will trigger all talos jobs for revision ' + revision + '!\n\nDo you want to proceed?')) {
+                return;
+            }
+
+            var times = window.prompt("Enter number of instances to have for each talos job", 6);
+            while (times > 6) {
+                times = window.prompt("We do not allow instances of each talos job to be greater than 6 times. Enter again", 6);
+            }
+
+            for (var iter=1; iter <= times; iter++) {
```

Comment:
I agree with @jmaher, we should add a parameter to indicate the number of retriggers.